### PR TITLE
adding the s to https

### PR DESCRIPTION
**Pull request by osbjmg**
and other odds and ends, along with a "timezone change" and some python3 compliance.
soon I'll need to validate the whole site again against html5, because, let's face it, there are some exceptions.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 $filename = 'STOCK_RT.json';
 $filepath = 'bin/'.$filename;
 $theTicker = 'ANET';
-$tzOffset = '04:00'; // EDT 4, EST 5
+$tzOffset = '05:00'; // EDT 4, EST 5
```
Comment (dathelen) [inline index.php]: would like to see this done dynamically but given how much you hate timezones I will allow this

File: bin/isanetbelow60.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
 import os
 import requests
 import json
 import re
-import urllib2
+#import urllib2
+import urllib.request
 import sys
 import time
 #import ystockquote
```
Comment (dathelen) [inline bin/isanetbelow60.py]: I feel we can just remove this commented line.  Yahoo is dead to you

File: index.php (hunk 1 above)
Comment (osbjmg) [reply index.php]: I agree, this is a little disappointing, I will continue to improve this.

File: bin/isanetbelow60.py (hunk 2 above)
Comment (osbjmg) [reply bin/isanetbelow60.py]: You're right, who needs them?

**Pull request merged**